### PR TITLE
Improves BoxStation supermatter

### PR DESCRIPTION
**Pull request by ACCount12**
* Emitter room is expanded. New reflectors allow some crazy emitter setups, and the old room was not big enough to be a playground.
* New pipe is added: Output to Waste. It allows you to redirect supermatter loop output into the scrubbers system by opening one valve and closing another. Useful if you don't want your gas to be wasted.

Before:
default

After:
default

### CODE REVIEW

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -57805,6 +57815,112 @@
...
+/area/engine/engineering)
+"Qoz" = (
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/obj/effect/turf_decal/stripes/line,
+/obj/machinery/atmospherics/components/binary/valve/open{
+	dir = 4;
+	name = "Output to Waste";
+	open = 0
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Why are you editing a var to do this when you used valve/open

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 2]
```diff
@@ -57805,6 +57815,112 @@
...
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/turf/open/floor/plasteel/black,
+/area/engine/engineering)
+"QoC" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden,
+/turf/open/floor/plasteel/black,
+/area/engine/engineering)
+"QoD" = (
+/obj/structure/table,
+/obj/item/stack/sheet/metal{
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: There should be a metal/fifty path too iirc

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 3]
```diff
@@ -57805,6 +57815,112 @@
...
+"QoH" = (
+/obj/item/wrench,
+/obj/item/weldingtool,
+/obj/item/clothing/head/welding{
+	pixel_x = -3;
+	pixel_y = 5
+	},
+/obj/structure/rack{
+	dir = 8;
+	layer = 2.9
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Layer edit?

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 4]
```diff
@@ -57805,6 +57815,112 @@
...
+/area/engine/engineering)
+"QoJ" = (
+/obj/machinery/light,
+/turf/open/floor/plating,
+/area/engine/engineering)
+"QoK" = (
+/obj/item/crowbar/large,
+/obj/structure/rack{
+	dir = 8;
+	layer = 2.9
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Layer?

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 5]
```diff
@@ -57805,6 +57815,112 @@
...
+/turf/open/floor/plating,
+/area/engine/engineering)
+"QoJ" = (
+/obj/machinery/light,
+/turf/open/floor/plating,
+/area/engine/engineering)
+"QoK" = (
+/obj/item/crowbar/large,
+/obj/structure/rack{
+	dir = 8;
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Changing dir doesn't do anything for racks iirc

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 6]
```diff
@@ -57805,6 +57815,112 @@
...
+/area/engine/engineering)
+"QoH" = (
+/obj/item/wrench,
+/obj/item/weldingtool,
+/obj/item/clothing/head/welding{
+	pixel_x = -3;
+	pixel_y = 5
+	},
+/obj/structure/rack{
+	dir = 8;
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Dir

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by ACCount12** [comment]
All fixed. I just copied objects from the map most of the time, looks like it was a bad idea.

**Comment by ACCount12** [comment]
I also removed dirs and layers from all the racks on the map, so this would not repeat.

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 7]
```diff
@@ -54708,39 +54598,46 @@
...
 	},
-/obj/machinery/meter,
-/obj/machinery/atmospherics/pipe/simple/scrubbers/visible{
-	dir = 4
+/obj/machinery/atmospherics/components/binary/valve/digital{
+	dir = 4;
+	icon_state = "dvalve_map";
+	name = "Output Release";
+	open = 1;
+	tag = "icon-dvalve_map (EAST)"
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Dirty var

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 8]
```diff
@@ -54708,39 +54598,46 @@
...
 "cGC" = (
 /obj/effect/turf_decal/stripes/line{
 	dir = 6
 	},
-/obj/machinery/meter,
-/obj/machinery/atmospherics/pipe/simple/scrubbers/visible{
-	dir = 4
+/obj/machinery/atmospherics/components/binary/valve/digital{
+	dir = 4;
+	icon_state = "dvalve_map";
```
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: Dirty var

Comment (ShizCalev) [reply _maps/map_files/BoxStation/BoxStation.dmm]: Also is there a /open path for this?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/BoxStation/BoxStation.dmm (hunk 8 above)
Comment (ACCount12) [reply _maps/map_files/BoxStation/BoxStation.dmm]: nope

**Comment by ACCount12** [comment]
"Dirty vars" removed, hope this is the last time.

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**